cmd/openshift-install/create.go

```diff
@@ -323,6 +326,13 @@ func waitForInitializedCluster(ctx context.Context, config *rest.Config) error {
 		},
 	)
 
+	// If we timed out and the CVO failed, print out the first failing operator
+	if err != nil && clusterVersion != nil {
+		if cov1helpers.IsStatusConditionTrue(clusterVersion.Status.Conditions, configv1.OperatorFailing) {
+			err = errors.New(cov1helpers.FindStatusCondition(clusterVersion.Status.Conditions, configv1.OperatorFailing).Message)
```

Comment:
`cov1helpers.FindStatusCondition` can return nil.

Reply:
nvm, the if prevents that. ignore :)

Reply:
We have this same pattern just above though; it can't return `nil` if the `cov1helpers.IsStatusConditionTrue` is true right?